frontend/src/component/insights/components/LineChart/LineChartComponent.tsx

```diff
@@ -126,7 +97,7 @@ const LineChartComponent: FC<{
                 key={cover ? 'cover' : 'chart'}
                 options={options}
                 data={data}
-                plugins={[customHighlightPlugin]}
+                plugins={[customHighlightPlugin({ width: 26 })]}
```

Comment:
The analytics line charts have a set width in their custom function, so we'll retain that here.